auth/src/api/userResources.js

```diff
@@ -0,0 +1,66 @@
+const router = require("express").Router();
+const User = require('../models/User');
+const jwt_decode = require('jwt-decode');
+
+// Get saved resources of a user
+router.get(
+  '/resources',
+  async (req, res) => {
+    const userId = jwt_decode(req.headers.token)['userId'];
+    const user = await User.findById(userId);
+    res.status(200).json({
+      code: 200,
+      message: `Succesfully got saved resources`,
+      success: true,
+      result: user.savedResources,
+    });
+  },
+);
+
+// Delete resource from the saved resources of user
+router.put(
+  '/deleteresource',
```

Comment:
This should probably just be changed to a `router.delete(/resource)` or something